dist/split-test.js

```diff
@@ -17,6 +17,8 @@
     }
 }(this, function() {
 
+    var DEFAULT_LIFETIME = 30 * 24 * 3600;
```

Comment:
Yup, IMO we should both to ms, since that's the actual desired unit. Also worth adding a 
comment to the splitTest function describing expected parameters and units, since Shawn mentioned people copy this into other places.